dev-ruby/googleapis-common-protos-types/googleapis-common-protos-types-1.6.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby30 ruby31"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Common protocol buffer types used by Google APIs"
+HOMEPAGE="https://github.com/googleapis/common-protos-ruby"
+SRC_URI="https://github.com/googleapis/common-protos-ruby/archive/${PN}/v${PV}.tar.gz -> ${P}.tar.gz"
+RUBY_S="common-protos-ruby-${PN}-v${PV}/${PN}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND=""
```

Comment:
This most likely does not do what you think: they reset the dependencies that were added by the eclass. If these are empty just leave them out altogether.